utils/pagination.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.pagination import PageNumberPagination as RFPageNumberPagination
+
+
+class PageNumberPagination(RFPageNumberPagination):
+    page_size = 10
+    max_page_size = 80
+    
+    def get_page_size(self, request):
+        if 'page_size' in request.query_params:
+            return min(self.max_page_size, int(request.query_params['page_size']))
```

Comment:
What if someone tries to supply `page_size` as a string? => `ValueError`

Use `"example string".isnumeric()` to check if it is a number before you try casting it 👍